README.md

```diff
@@ -33,7 +33,7 @@ to 5.3.8 (see README.PHP-5.3.9-and-5.3.10).
 CI Server
 ---------
 Take a look at the Jenkins CI Server (http://ci.marcelog.name/) to check out
-the docs, metrics, and pear and phar packages available.
+the docs, metrics.
```

Comment:
`the docs and metrics`

As an aside: last time I looked, it seemed that the CI server was down.